configs/default/JHEF-JHEF405PRO.config

```diff
@@ -100,9 +118,9 @@ set beeper_inversion = ON
 set beeper_od = OFF
 set system_hse_mhz = 8
 set max7456_spi_bus = 3
-set dashboard_i2c_bus = 1
 set flash_spi_bus = 3
 set gyro_1_bustype = SPI
 set gyro_1_spibus = 1
-set gyro_1_sensor_align = CW180
-set gyro_1_align_yaw = 1800
+set gyro_1_sensor_align = CW90
+set gyro_1_align_yaw = 900
+set gyro_2_spibus = 1
```

Comment:
Do we have a dual gyro ?

Reply:
Yes. We have a dual gyro. Thanks @haslinghuis I saw your modifications, so I made the changes without rebase. I saw that your modifications were correct and can be merged.